drivers/SmartThings/matter-lock/fingerprints.yml

```diff
@@ -1,3 +1,11 @@
+matterManufacturer:
+#Nuki
+  - id: "Nuki Smart Lock Pro"
+    deviceLabel: Nuki Smart Lock Pro
+    vendorId: 0x135d
+    productId: 0xb1
+    deviceProfileName: lock-without-codes-or-tamper
```

Comment:
Make sure this is updated to match the new profile name from https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/868 (`lock-nocodes-notamper`) :)